CHANGELOG.md

```diff
@@ -1,9 +1,41 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+This project adheres to [Semantic Versioning](http://semver.org/)
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## 1.2.0 2017-05-13
+## [Unreleased]
+
+## [2.0.0] - 2018-02-06
+### Added
+- Initial support for OpenBSD! (@smbambling)
+- Ubuntu now get's `apt-transport-https` installed to support HTTPS repos. (@kevit)
+- Default to HTTPS APT repo's.  @jaredledvina
+- Allow for configuring when a node gets the `sensu-client` config file. (@tculp)
+- Allow for deploying client definitions based on groups. (@tculp)
+- Default to HTTPS Yum repo's and install the Yum key for package signing validation via HTTPS.  (@jaredledvina)
+- Used HTTPS for APT key.  (@jaredledvina)
+- Amazon Linux now get's proper yum repo and supports Amazon Linux 2. (@romainrbr)
+- Yum based distros now get EPEL to support installing a newer and supported version of RabbitMQ. (@romainrbr)
+- CentOS now supports using Bintray mirrors for installing RabbitMQ to work around Erlang issues with older versions. (@romainrbr)
+- All PR's are now required to pass TravisCI tests.  (@jaredledvina)
+- Ensure that we configure the `mode` and `umask` for files to work in a more restrictive environment. (@roumano)
+- Debian and Ubuntu switch to Bintray for RabbitMQ to match yum distros. (@jaredledvina)
+
+### Changed
+- Switched from Gitter to `#ansible` in the Sensu Community Slack. (@grepory)
+- Bumped SSL tools version to 1.2 by default. (@marji)
+- Update 'Generate SSL Certs' to support Ansible 2.4. (@tculp)
+
+### Fixed
+- Ubuntu - Redis now always binds to `0.0.0.0` to match other distro's configurgation and to ensure it's accessible. (@tculp)
+
+### Breaking Change
```

Comment:
This should always be the first section under the version/unreleased because it is the most important for users to quickly spot and evaluate.